.circleci/config.yml

```diff
@@ -0,0 +1,326 @@
+version: 2.1
```

Comment:
I had previously deleted circleci files since we use GitHub actions

Reply:
Agreed.